demisto_sdk/commands/find_dependencies/tests/find_dependencies_test.py

```diff
@@ -1119,6 +1120,33 @@ def test_search_packs_by_items_names_or_ids(item_names, section_name, expected_r
 
 
 class TestDependencyGraph:
+    @pytest.mark.parametrize('source_node, expected_nodes_in, expected_nodes_out',
+                             [('1', ['1', '2', '3'], ['4']),
```

Comment:
i think it would be better if we use pack names here instead of numbers, just so it would be more intuitive